WordPress/src/main/java/org/wordpress/android/models/Post.java

```diff
@@ -54,8 +54,8 @@
     private String quickPostType;
     private PostLocation mPostLocation;
 
-    private int lastKnownRemoteFeaturedImageId;
-    private int featuredImageId = FEATURED_IMAGE_INIT_VALUE;
+    private long lastKnownRemoteFeaturedImageId;
+    private long featuredImageId = FEATURED_IMAGE_INIT_VALUE;
```

Comment:
Minor nit: `FEATURED_IMAGE_INIT_VALUE` should be changed from an `int` to a `long`.